src/IssueViz.Security/Taint/TaintIssueToIssueVisualizationConverter.cs

```diff
@@ -53,21 +53,31 @@ public IAnalysisIssueVisualization Convert(SonarQubeIssue sonarQubeIssue)
             var analysisIssue = ConvertToAnalysisIssue(sonarQubeIssue);
             var issueViz = issueVisualizationConverter.Convert(analysisIssue);
 
+            CalculateLocalFilePaths(issueViz);
+
             return issueViz;
         }
 
+        private void CalculateLocalFilePaths(IAnalysisIssueVisualization issueViz)
+        {
+            var allLocations = issueViz.GetAllLocations();
+
+            foreach (var location in allLocations)
+            {
+                location.CurrentFilePath = absoluteFilePathLocator.Locate(location.Location.FilePath);
+            }
+        }
+
```

Comment:
Presumably we're going to need to make the same change in the hotspot converter when we get round to handling hotspots with secondary locations? If so, it would be useful if we could create a ticket for it so we don't forget.

Or could the work be pushed to the `IAnalysisIssueVisualizationConverter`?

Reply:
It could create a performance hit for `IAnalysisIssueVisualizationConverter` - it is used by live issues where the CurrentFilePath is exactly the same as FilePath (since it was just calculated).
I will create a ticket for hotspots.